concepts/default-enarx-trust-process.md

```diff
@@ -0,0 +1,480 @@
+# TBD: Default Enarx trust process
+- Authors: [Mike Bursell](mike@p2ptrust.org)
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2020-03-17
+- Status Note: under discussion
+- Start Date: 2020-03-17 (date you started working on this idea)
+- Tags: trust
+- Addresses: (Issue #)[https://github.com/enarx/enarx/issues/]
+
+TODO: this is curently listed as a "concepts" document, but also
+includes design information and implementation requirements.  Need
+to decide where it best sits.  Neither "concepts" nor "features"
+(as described in the README.md) fit very well.
+
+## Summary
+
+Setting up trust domains and being able to transfer certain entities
+between them in central to Enarx.  This RFC describes the default
+trust process for Enarx and lays out sets of requirements for
+implementations.
+
+## Motivation
+
+One of the key aims of the Enarx project is to allow clients to run
+workloads on hosts which are "untrusted".  This RFC builds on previous
+RFCs to describe the default model by which Enarx allows this to take
+place, using the capabilities of TEEs, the underlying CPU(s) and
+firmware of the host and other parties.
+
+It should be noted that the key party for whom trust is considered
+important within the Enarx project is the workload owner - in this case,
+the tenant.  We should remember that all trust relationships are
+explicitly uni-directional (though this does not preclude a
+corresponding trust relationship in the other direction).  In the case
+of Enarx, it is trust relationships from the workload entity to other
+entities (primarily the host owner in this document), and maintenance
+of the tenant's trust domain, which are always the priority - any
+other benefits which may accrue to other parties, are considered
+side benefits.
+
+On a different note, this RFC deals mainly with the communications
+between the various Enarx components.  Although the Orchestrator is
+involved and mentioned in this document, it is not the document's focus.
+
+## Trust state transitions in Enarx
+
+See rfc#0000x for an introduction to trust relationships.
+See rfc#0000x for an introduction to trust domains and their application
+to Enarx.
+See rfc#0000x for an introduction to endorsing authorities, trust anchors
+and trust pivots.
+
+rfc#0000x described the following Enarx use case states for the default
+Enarx trust model (TODO: check correctness in rfc#0000x and normalise):
+ - State 0 - initial state
+ - State 1 - pre-attestation
+ - State 2 - attested
+ - State 3 - provisioned
+
+Non-Enarx use cases were also discussed, but these are irrelevant to this
+discussion, and therefore ignored in this RFC.
```

Comment:
Same comment as above, let's remove this information cited from another RFC.

Reply:
It was intended to set the scene.  If we _don't_ have references to other RFCs, how do you know to what to refer?  I don't think you can have it both ways...